paper/main.tex

```diff
@@ -880,14 +892,16 @@ \section{Evaluation of performance}\label{section:evaluation_of_performance}
 The distributions of the \(CD\) to \(C\) rate evaluate the behaviour of a
 successful strategy after its opponent has defected against it. In standard
 tournaments it was observed that a successful strategy reciprocates with a
-probability of 0.5.
-This leads to the recommendation of needing to be somewhat provocable.
-This is distinct between the tournament types. In
+probability of 0.5, and in a setting that the type
+of the tournament can vary between all the examined types a winning strategy
+would reciprocate on average with a probability of 0.58. In
 tournaments with noise a strategy is less likely to cooperate following a
 defection compared to standard tournaments, and in probabilistic ending
-tournaments a strategy will reciprocate a defection. In a setting that the type
-of the tournament can vary between all the examined types a winning strategy
-would reciprocate on average with a probability of 0.58.
+tournaments a strategy will reciprocate a defection.
+This leads to revisiting the recommendation of being provocable to defections made
+by R. Axerlod. In fact only in tournaments with short matches should a strategy
```

Comment:
Is that quite right? In short matches strategies should be less nice and highly provocable I think, i.e. simply more aggressive over all. Mutual cooperation is build by history and the expectation of future interactions. From a purely payoff point of view, if one knows that an interaction will end soon then defecting could result in a higher payoff over the last few interactions. It is uncertainly about when interactions will end and the desire to avoid (or impact of) prolonged mutual defection that encourages mutual cooperation.

Reply:
I agree with you Marc. I think that's what I am saying.

> In fact only in tournaments with short matches should a strategy
be provocable

and by provocable we mean deterministically just defect after receiving a defection.

Reply:
I think I'm just struggling with "provocable" meaning "completely provocable" rather than "somewhat provocable" depending on the environmental conditions, such as noise, or expectation of a game ending soon.

You might enjoy [this article](https://grantland.com/features/diplomacy-the-board-game-of-the-alpha-nerds/) about the game diplomacy. Near the end there is a tale of a guy that spends years developing a reputation as a never-defecting alliance member, but when given a chance at a very rare total victory in a world championship, he betrays his compatriots at the last instant.

Reply:
I am definitely going to read that one!

Reply:
Do you think https://github.com/Nikoleta-v3/meta-analysis-of-prisoners-dilemma-tournaments/pull/30/commits/2d6e502c5de18bfd720ed69c09f4eb405bef1f06 makes our suggestion clearer?

Reply:
It's closer, but sort of implicitly suggests now to not be provocable or generous in "standard" matches. I think it's more accurate to say be somewhat provocable and somewhat generous, how much so depends on average game length, noise, and the population of opponents. I'll see if I can find a way to say that succinctly.

Reply:
Okay.

 Would it be easier for you if I merge the current `PRs` in?

Reply:
> You might enjoy [this article](https://grantland.com/features/diplomacy-the-board-game-of-the-alpha-nerds/) about the game diplomacy. Near the end there is a tale of a guy that spends years developing a reputation as a never-defecting alliance member, but when given a chance at a very rare total victory in a world championship, he betrays his compatriots at the last instant.

Brilliant!

Reply:
Sure go ahead and merge